paper.tex

```diff
@@ -0,0 +1,223 @@
+\documentclass[format=acmsmall]{acmart}
+\begin{document}
+\title{GPU-friendly Stroke Expansion}
+\author{Raph Levien}
+\author{Arman Uguray}
+\affiliation{
+    \institution{Google}
+    \city{San Francisco}
+    \state{CA}
+    \country{USA}
+}
+
+\begin{abstract}
+    Vector graphics includes both filled and stroked paths as the main primitives. While there have been many proposed techniques for rendering filled paths on GPU, stroked paths have proved more elusive. This paper presents a technique for performing stroke expansion, namely the generation of an outline representing the stroke of the given input path, an operation which is considered global, but which we nonetheless implement using a fully parallel algorithm. Our method relies on several novel techniques, including an encoding of the input suitable for parallel decoding and processing, an Euler spiral based method for computing flattened parallel curves. While we focus on flattened output consisting of polylines, the methods can be adapted to preserving curves in the output as well.
```

Comment:
Perhaps we should conclude the abstract with an assertion of some of the accomplishments, e.g.:

_Our method produces a near-optimal number of line segments for the filled outline of a stroked cubic Bézier and can be fully evaluated on a GPU with minimal preprocessing._

I considered adding a "_supports all common cap and join styles_" to that sentence but I'm not entirely sure if that needs to be mentioned in the abstract.